src/utils/data-utils.ts

```diff
@@ -217,6 +217,49 @@ interface ColumnDataShape {
   units?: string;
 }
 
+interface LevelMap {
+  [key: string]: { description: string; standardizedTerm: string };
+}
+
+function buildCategoricalLevels(
+  allValues: string[] | undefined,
+  missingValues: string[] | undefined,
+  dictLevels: { [key: string]: { Description: string; TermURL?: string } } | undefined,
```

Comment:
Why do you need to expand these parameters? They are all part of the `column` object, no? I'd say that's a good thing and keeps the function signature focused on the important parts.

Reply:
Will pass a group object of column, columnData, and standardizedTerms, `{ column, columnData, standardizedTerms }` and we can destructure internally.